store_app/business/ConnectExternalAPI.py

```diff
@@ -0,0 +1,32 @@
+from urllib.request import urlopen
+import json
+
+
+def get_poster_link(imdb_id):
+    # movie = movie_id
+    # add = movie.replace(' ', '+')
+    url = "https://www.omdbapi.com/?i=" + imdb_id + '&apikey=aafb7441&'
+    data = urlopen(url).read()
+    data_p = json.loads(data)
+    image_link = data_p['Poster']
+    return image_link
+
+
+def get_movie_description(imdb_id):
+    url = "https://www.omdbapi.com/?i=" + imdb_id + '&apikey=aafb7441&'
```

Comment:
The url construction is repeated multiple times, you could have a small private helper function that does that once and use it in all these methods.